src/types/mod.rs

```diff
@@ -424,6 +426,128 @@ impl ValueInternal for JsNumber {
     }
 }
 
+/// A JavaScript Date object.
+#[repr(C)]
+#[derive(Clone, Copy, Debug)]
+#[cfg(feature = "napi-5")]
+pub struct JsDate(raw::Local);
+
+#[cfg(feature = "napi-5")]
```

Comment:
I think it will be cleaner to crate a `date.rs` file that is conditionally declared instead of adding the attribute in many places.

Reply:
+1, good idea!